vhost_user_fs/src/filesystem.rs

```diff
@@ -1043,6 +1045,27 @@ pub trait FileSystem {
         Err(io::Error::from_raw_os_error(libc::ENOSYS))
     }
 
+    /// Setup a mapping so that guest can access files in DAX style.
+    #[allow(clippy::too_many_arguments)]
+    fn setupmapping<T: FsCacheReqHandler>(
+        &self,
+        _ctx: Context,
+        inode: Self::Inode,
+        handle: Self::Handle,
+        foffset: u64,
+        len: u64,
+        flags: u64,
+        moffset: u64,
+        vu_req: &mut T,
+    ) -> io::Result<()>;
+
+    fn removemapping<T: FsCacheReqHandler>(
+        &self,
+        _ctx: Context,
+        requests: Vec<RemovemappingOne>,
+        vu_req: &mut T,
+    ) -> io::Result<()>;
```

Comment:
All the other methods in this trait provide default implementations (most just return ENOSYS) so that types that implement this trait are not required to implement those methods.  Not providing a default implementation here means that every type needs to provide an implementation of this method even if they don't support it.

I'd recommend providing the default ENOSYS implementation for both of these as well:

```rust
fn removemapping<T: FsCacheReqHandler>(
    &self,
    _ctx: Context,
    requests: Vec<RemovemappingOne>,
     vu_req: &mut T,
) -> io::Result<()> {
    Err(io::Error::from_raw_os_error(libc::ENOSYS))
}
```

Reply:
Sure, will update it.